### PR TITLE
Add LibreTube icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes none

**Popularity metric:**

It has ~8k stars on GitHub. See https://github.com/libre-tube/LibreTube

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Used the accent color from the website. Also, I just discovered that they have a monochromatic icon version in their repository. Should this be used instead? https://github.com/libre-tube/libre-tube.github.io/blob/main/assets/icons/monochrome.svg

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8791,6 +8791,11 @@
                 "type": "MPL-2.0"
             }
         },
+        {
+            "title": "LibreTube",
+            "hex": "FFBBAA",
```
Comment (uncenter) [inline _data/simple-icons.json:8796]: I would go with `#FF9699`, using https://meyerweb.com/eric/tools/color-blend/#FF5A8B:FFD1A7:1:hex from https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines. That accent color specifically isn't present in the color logo, and I think this darker red is more fitting just as far as the eye goes.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8791,6 +8791,11 @@
...
+            "source": "https://github.com/libre-tube/libre-tube.github.io/blob/e5e10090cab71ee7c0abdfbf2789977025733eb7/assets/icons/LibreTube.svg"
```
Comment (uncenter) [inline _data/simple-icons.json:8797]:
```suggestion
            "source": "https://github.com/libre-tube/libre-tube.github.io/blob/e5e10090cab71ee7c0abdfbf2789977025733eb7/assets/icons/icon.svg"
```

The icon source still matches (perfectly!) with your icon here, which I guess is the LibreTube icon minus the background of the source you have linked here now.

File: icons/libretube.svg
Comment (uncenter) [inline icons/libretube.svg:1]: Looks good!

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (CodeShakingSheep) [reply _data/simple-icons.json]: Agreed. I just updated the color hex code.

**Review by uncenter — APPROVED** [review]

**Pull request merged**